src/targets/common.ini

```diff
@@ -7,7 +7,7 @@ framework = arduino
 extra_scripts =
 	pre:python/build_flags.py
 	python/build_env_setup.py
-monitor_speed = 420000
+monitor_speed = 460800
```

Comment:
This one effects to all targets. That is set to 420k for a reason that we have more RX targets than TX targets. So don't change this line or you have to modify all RX targets :)

ESP32 monitor speed was set in:
```
[env_common_esp32]
monitor_speed = 460800
```

but it seems that @JyeSmith removed it in PR #453 . You just need to revert that one line and all good :)

Reply:
it also affects STM based TX targets as well, adding this to each of those targets individually would be a bit of a repetition wouldn't it?

Reply:
you are right, I would do something like this for STMs:

```
[env_common_stm32_TX]
extends = env_common_stm32
monitor_speed = 460800
```

and then modify TX envs to use that, e.g.
```
[env:Frsky_TX_R9M_via_STLINK]
extends = env_common_stm32_TX
```

Reply:
This was reported for a Ghost TX, so will need to work for STM too